app/views/users/show.html.erb

```diff
@@ -1,13 +1,36 @@
-<!DOCTYPE html>
-<html>
-<head>
-<title>Blog App</title>
-<meta name="viewport" content="width=device-width, initial-scale=1">
-</head>
-<body>
-  <h2>User Details</h2>
-  <h4>Name: <%= @user.name %></h4>
-  <p>Bio: <%= @user.bio %></p>
-  <h5>Total Posts: <%= @user.posts_counter %></h5>
-</body>
-</html>
+ <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag "user", "data-turbo-track": "reload" %>
+    <%= javascript_importmap_tags %>
+<%= render "shared/header" %>
+<div class="container_user">
+  <div class="user-info">
+     <%= image_tag @user.photo_url, alt: @user.name %>
+     <div class = "user-info-2">
+    <h3>Name: <%= @user.name %></h3>
+        <h5>Total Posts: <%= @user.posts_counter.inspect %></h5>
+    <h4>Bio:</h4><p><%= @user.bio %></p>
+    </div>
+    </div>
+       <div class = "post-info">
+     <ul>
+  <% @user_posts.each do |post| %>
+    <li class = "post-border">
```

Comment:
- [ ] It would be nice if you use the `recent_posts` method here which you have made in the user model to show only the most three recent posts to make your project more automated 👍